setup.cfg

```diff
@@ -5,7 +5,7 @@ test=pytest
 python_files = tests.py test_*.py tests_*.py *_tests.py
 env =
 	KQUEEN_CONFIG_FILE=config/test.py
-	prometheus_multiproc_dir=/tmp/prometheus_client/
+	D:prometheus_multiproc_dir=/tmp/prometheus_client/
```

Comment:
What is this?

Reply:
prevent overriding env variables for pytest to avoid issues related to directory doesnot exist:
FileNotFoundError: [Errno 2] No such file or directory: '/tmp/prometheus_client/gauge_all_5850.db'
it happens when setup.cfg overrides env var for pytest run

Reply:
for ex https://travis-ci.org/Mirantis/kqueen/jobs/352239789
@atengler